include/envoy/network/filter.h

```diff
@@ -448,5 +457,14 @@ class FilterChainFactory {
                                             UdpReadFilterCallbacks& callbacks) PURE;
 };
 
+class DrainingFilterChains {
```

Comment:
I don't see this used in this PR. Please remove.

Reply:
Agree... Removing it and create the next PR soon